common/colour/colour.go

```diff
@@ -0,0 +1,100 @@
+package colour
```

Comment:
On the whole in code,  American spellings are prefered and we try and strictly adhere to this in Fn. - s/colour/color/